Lib/unittest/mock.py

```diff
@@ -3004,13 +3004,32 @@ def __get__(self, obj, obj_type=None):
     def __set__(self, obj, val):
         self(val)
 
+
+_timeout_unset = sentinel.TIMEOUT_UNSET
```

Comment:
Nice! Although I think `None` works just as well here:
"What timeout is set?"
"None, it blocks forever"

Reply:
What I was trying to achieve is a user changing the class timeout to be `3` then passing a timeout of `None` to mean "block". If this is None, how do we distinguish a user not passing anything and we getting the default vs they specifically saying "block"?

Reply:
Sure, that makes sense :-)